### PR TITLE
Improve default project structure

### PR DESCRIPTION
**Pull request by nqbao**
* Improve project structure accordingly to @TgithubJ  feedback
* Generate a default notebook (disable for now)
* Generate a default modeling script

Probably in the future, we will switch to yeoman or other generator instead of coding our own as of right now.

### CODE REVIEW

**Comment by ctn** [comment]
Do you want a review or are you saying this is still in-flight (created PR then changed your mind?)

**Comment by nqbao** [comment]
This is still in flight, and will need some clean up before reviewing/merging. I sent the PR to see the testing status.

I will probably ask @thangtp  or @khoama to review when it's ready.

**Comment by nqbao** [comment]
This is ready for review @thangtp

File: h1st/cli/project.py [hunk 1]
```diff
@@ -41,18 +42,20 @@ def new_model_cli(model_name):
...
         # TODO: try to figure out the project folder if possible
 
-        if not model_path.exists():
+        if not (path / "config.py").exists():
             raise ValueError('Please run this command in the package folder')
 
+        model_path.mkdir(exist_ok=True)
+
         class_name, module_name = _clean_name(model_name)
+        project_package = path.name
```
Comment (thangtp) [inline h1st/cli/project.py:51]: project_package: unused

File: h1st/cli/templates/model.txt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import h1st as h1
+import config
+
+class $$MODEL_CLASS$$(h1.MLModel):
+    def __init__(self):
+        # Instantiate your base (tensorflow, sklearn, pytorch, etc.) model here
+        # self.base_model = xxx
+        pass
+
+    def load_data(self) -> dict:
```
Comment (thangtp) [inline h1st/cli/templates/model.txt:15]: should we have .explore() here too?

**Comment by nqbao** [comment]
updated @thangtp

**Review by thangtp — APPROVED** [review]

**Pull request merged**